src/System.Private.ServiceModel/tools/PRService/pr.ashx

```diff
@@ -156,7 +199,13 @@ public class PullRequestHandler : IHttpHandler
         return success;
```

Comment:
The `return success` in `catch {...}` above can be removed (GitHub does not allow me to put comment there).